activerecord/lib/active_record/railtie.rb

```diff
@@ -276,5 +277,34 @@ class Railtie < Rails::Railtie # :nodoc:
         self.signed_id_verifier_secret ||= -> { Rails.application.key_generator.generate_key("active_record/signed_id") }
       end
     end
+
+    initializer "active_record_encryption.configuration" do |app|
+      config.before_initialize do
+        ActiveRecord::Encryption.configure \
+           master_key: app.credentials.dig(:active_record_encryption, :master_key) || ENV["ACTIVE_RECORD_ENCRYPTION_MASTER_KEY"],
```

Comment:
This is going to be the first time we will use env environments for configs. Why we don't put inside the configuration files?

Reply:
Addressed in https://github.com/basecamp/rails/commit/842734270bf63c9ce2a24332647210186d58253a